jbmc/src/java_bytecode/README.md

```diff
@@ -883,4 +883,27 @@ To be documented.
 
 \subsection java-trace-string-assignments String assignments
 
-To be documented.
+"Strings" only exist in Java. This section refers to the `java.lang.String`
+object representation in GOTO. It can take one of the following forms:
+
+ - Most strings are `java.lang.String` objects with a data field assigned to a
```

Comment:
:question: Can we be more precise than most?

Reply:
Is it precisely the dynamic (i.e. not literals in the source code) strings?

Reply:
Literals are also java.lang.Strings